waku/v2/protocol/waku_swap/waku_swap_types.nim

```diff
@@ -20,8 +20,16 @@ type
     amount*: uint32
     signature*: seq[byte]
 
+  BalanceReport* = object
+    totalCredit*: int
+    totalDebit*: int
+    balance*: int
+  
+
+
   CreditHandler* = proc (peerId: PeerId, amount: int) {.gcsafe, closure.}
   DebitHandler* = proc (peerId: PeerId, amount: int) {.gcsafe, closure.}
+  BalanceHandler* = proc() : BalanceReport {.gcsafe, closure.}
```

Comment:
What purpose does this handler serve? Vs just calling it directly, like accounting table

Reply:
Used the Balance function so there will Be room to add some logic before returning the data to the client. But that won't be b=necessary if I just use the accounting table directly.